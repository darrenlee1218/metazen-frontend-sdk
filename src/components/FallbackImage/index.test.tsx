import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { FallbackImage } from '.';
import { TestProviders } from '@utils/testing/TestProviders';

afterEach(cleanup);

// TODO: Very hard to test
it.todo('should display fallbackSrc if there is an error loading the original image');

it('matches snapshot', () => {
  const { container } = render(<FallbackImage src="404.png" fallbackSrc="fallback.png" />, { wrapper: TestProviders });
  expect(container).toMatchSnapshot();
});
