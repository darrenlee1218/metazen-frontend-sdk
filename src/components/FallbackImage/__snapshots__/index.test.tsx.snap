// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <img
    class="MuiBox-root css-0"
    src="404.png"
  />
</div>
`;
